nautobot/docs/plugins/development.md

```diff
@@ -1037,19 +1037,19 @@ class YourAppModelUIViewSet(NautobotUIViewSet):
 
 **One caveat of using the NautobotUIViewSet is that the `queryset`, `serializer_class` and `table_class` attribute of the `YourAppModelUIViewSet` has to be set before most of the `NautobotUIViewSet` functionalities will become available.**
 
-By default the URL patterns generated by a `NautobotUIViewSet` are based on the model's `slug` (`/model-name/<slug>/` for the detail view, `/model-name/<slug>/edit/` for the edit view, etc.). If your model lacks a `slug` field, or if you otherwise need to use a different field to look up an object, just override the default `lookup_field` in your ViewSet attributes:
+By default the URL patterns generated by a `NautobotUIViewSet` are based on the model's `pk` (`/model-name/<pk>/` for the detail view, `/model-name/<pk>/edit/` for the edit view, etc.). if you need to use a different field to look up an object, just override the default `lookup_field` in your ViewSet attributes:
 
 ```python
 from nautobot.apps.views import NautobotUIViewset
 
 class YourAppModelUIViewSet(NautobotUIViewSet):
     ...
-    lookup_field = "pk"
+    lookup_field = "slug"
     ...
 ```
 
```

Comment:
````suggestion
```

+/- 2.0.0
    The default `lookup_field` for `NautobotUIViewSet` has been changed from `"slug"` to `"pk"`.
````